packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -0,0 +1,122 @@
+import { CSSRules } from "@microsoft/fast-jss-manager";
+import { DesignSystem } from "../design-system";
+import { format, toPx } from "@microsoft/fast-jss-utilities";
+import { focusOutlineWidth, outlineWidth } from "./design-system";
+
+export const highContrastSelector: string = "@media (-ms-high-contrast:active)";
+
+export enum HighContrastColor {
+    text = "WindowText",
+    hyperLinks = "LinkText",
+    disabledText = "GrayText !important",
```

Comment:
is `!important` necessary for only this value? What about the others?

Reply:
Disabled should always trump all other values, when it is being used, in the style. So it always gets an !important, here, instead of having to repeat it in the const below.